modules/social_features/social_album/social_album.install

```diff
@@ -53,29 +52,14 @@ function social_album_install() {
 
   module_set_weight('social_album', 5);
 
-  _social_album_blocks('_social_album_install_blocks');
-}
-
-/**
- * Add new paths where blocks should be displayed.
- *
- * @param string $pages
- *   The current sequence of block paths.
- * @param array $paths
- *   List of blocks which should be updated and paths which should be added.
- *
- * @return string
- *   List of block paths with new paths.
- */
-function _social_album_install_blocks($pages, array $paths) {
-  return $pages . "\r\n" . implode("\r\n", $paths);
+  _social_album_blocks();
 }
 
 /**
  * Implements hook_uninstall().
  */
 function social_album_uninstall() {
-  _social_album_blocks(function ($pages, array $paths) {
+  _social_album_blocks(NULL, function ($pages, array $paths) {
```

Comment:
I'd say that the function from 2nd argument should be moved to the core module because it might be re-used.